src/MLOps.NET.SQLite.IntegrationTests/SQLiteMetaDataStore.cs

```diff
@@ -0,0 +1,46 @@
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace MLOps.NET.SQLite.IntegrationTests
+{
+    [TestClass]
+    public class SQLiteMetaDataStore
+    {
+        [TestMethod]
+        public void CreateExperimentAsync_Success()
+        {
+            //Arrange
+            MLLifeCycleManager mlm = new MLLifeCycleManager();
```

Comment:
Super awesome that you're adding the first tests! For a convention, I think it would be nice if we call the class we are testing either classUnderTest or unitUnderTest, I'll let you decide between those two :)

Reply:
I'm not sure this is what you meant, I put CreateExperimentAsync_Always_ReturnsNonEmptyGuid and UploadModelAsync_ValidModelPath_UploadSuccess